bbb-install.sh

```diff
@@ -111,6 +111,9 @@ main() {
   LETS_ENCRYPT_OPTIONS="--webroot --non-interactive"
   SOURCES_FETCHED=false
 
+  # If grep'ing for a key in "apt-key list" is dangerous, should we find a better way to do it?
```

Comment:
@XSpielinbox is right.  Since this is supposed to be a simple cleanup PR, I dropped the key management stuff completely.  I'll put something better in a separate PR.